.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,28 @@
-## Summary
 
-*Short description*
+# Checklist
 
-## TODO
+- [ ] Target milestone
+- [ ] New tests
+- [ ] Updated documentation
+- [ ] External dependency updates
+- [ ] Internal dependency updates (with link to diff)
+- [ ] Screenshots
```

Comment:
Doesn't this tick itself kind of? If screenshots are relevant, people can add them in the deleted headline / summary below.

Reply:
this is for front-end PRs - not issues